codalab/worker_manager/worker_manager.py

```diff
@@ -247,3 +252,19 @@ def run_one_iteration(self):
             logger.info('Starting a worker!')
             self.start_worker_job()
             self.last_worker_start_time = time.time()
+
+    def filter_bundles(
+        self, bundles: List[Dict[str, Dict[str, Union[int, str]]]]
+    ) -> List[Dict[str, Dict[str, Union[int, str]]]]:
```

Comment:
Can we define a type alias for `List[Dict[str, Dict[str, Union[int, str]]]]` just to make this a bit cleaner?